src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/VSCompletionList.cs

```diff
@@ -14,6 +15,8 @@ protected VSCompletionList(CompletionList innerCompletionList) : base (innerComp
 
         public Container<string> CommitCharacters { get; set; }
 
+        public object Data { get; set; }
```

Comment:
This name seems too basic. What kind of data may be on here, is it just two possible types or is it totally the wild west? In the absence of a more descriptive name a doc comment seems appropriate.

Reply:
Totally wild west, just matters that it's serializable 😄. This corresponds with LSP's CompletionItem.Data
![image](https://user-images.githubusercontent.com/2008729/115597971-be7f4680-a28e-11eb-9358-3a5807f383e6.png)

Reply:
Maybe just steal that comment for inclusion here then?

Reply:
Sure can do!